emacs/merlin-imenu.el

```diff
@@ -58,6 +61,12 @@
              (setq merlin-imenu--value-list (cons marker merlin-imenu--value-list)))
             ((string= kind "Type")
              (setq merlin-imenu--type-list (cons marker merlin-imenu--type-list)))
+            ((string= kind "Module")
+             (setq merlin-imenu--module-list (cons marker merlin-imenu--module-list)))
+            ((string= kind "Modtype")
```

Comment:
If I'm reading the output of `(merlin-call "outline")` correctly, I think `"Modtype"` should be `"Signature"`. The entries for `"Module"` and `"Class"` look correct though.

Reply:
I can confirm that your reading is correct. Fixed now.